docs/eigenlayer/avs-guides/rewards.md

```diff
@@ -31,4 +31,18 @@ Integration Notes:
 - Operators will only be distributed rewards on **entire** days that they have opted into the AVS.
 - Due to the rounding in the off-chain process, we recommend not making range submission token amounts with more than 15 significant digits of precision. If more than 15 significant digits are provided, the extra precision will be truncated.
 - Rewards can be made in multiple ERC-20 tokens by calling `createAVSRewardsSubmission` for each ERC-20 token to reward in.  
-- There are several requirements for successfully calling `createAVSRewardsSubmission`. It's recommended to read further details [here](https://github.com/Layr-Labs/eigenlayer-contracts/blob/v0.3.0-rc3-holesky-preprod-rewards/docs/core/RewardsCoordinator.md#createavsrewardssubmission).
\ No newline at end of file
+- There are several requirements for successfully calling `createAVSRewardsSubmission`. It's recommended to read further details [here](https://github.com/Layr-Labs/eigenlayer-contracts/blob/v0.3.0-rc3-holesky-preprod-rewards/docs/core/RewardsCoordinator.md#createavsrewardssubmission).
+
+## FAQ
+
+### When are rewards submissions including in daily calculation
+
+An AVS's reward submission is included in the calculation 2 days after it is submitted. For example, if the AVS submits a rewards on August 2nd, it will be included in the August 4th rewards calculation.
```

Comment:
```suggestion
An AVS' reward submission is included in the calculation 2 days after it is submitted. For example, if the AVS submits a rewards submission on August 2nd, it will be included in the August 4th rewards calculation.
```